test/test_core.rb

```diff
@@ -721,22 +721,6 @@ def test_bad_content
       end
     end
 
-    def test_iso_8859_1_output_function
```

Comment:
Why do we need remove this test?

Reply:
I was testing `Source#scan`, which is no longer needed, so I removed it as unnecessary.
However, since the purpose seemed to be to test iso-8859-1, it was not appropriate to remove it. Sorry.

Replace `Source#scan` in the test with `Source#match` and the test will work, so modify it in a way that leaves the test.